monkey/agent_plugins/credentials_collectors/chrome/src/chrome_browser_local_state.py

```diff
@@ -0,0 +1,56 @@
+import base64
+import json
+import logging
+from pathlib import PurePath
+from typing import Collection, Optional, Set
+
+logger = logging.getLogger(__name__)
+
+
+class ChromeBrowserLocalState:
```

Comment:
Consider flattening this class into the local data class. This class is effectively some class/level boilerplate wrapped around a parsing function. We could just move the parsing function into the local data class and reduce the overall number of classes/modules in play.